csharp/device/samples/UWPSample/project.lock.json

```diff
@@ -11523,7 +11523,7 @@
       ]
     },
     "Microsoft.ApplicationInsights.WindowsApps/1.0.0": {
-      "sha512": "NvBQnFeiFd0O1QdBz06UGApD7zn7ztVi7qO18IsM3EjiXRNgfrEBXB+azNm8XqLY8xGFAqh3HAuSd/wHZMe0XA==",
```

Comment:
If it's autogenerated, there's no need to check them in and review the changes. I'll remove them when we merge the PR.